addons/website/static/src/less/website.snippets.less

```diff
@@ -11,6 +11,15 @@
     }
 }
 
+// Inner content Snipptes
+#wrapwrap
+    .row
+        [class^="col-md-"]
+            .s_auto_height
+            {
+                height: auto;
+            }
```

Comment:
I don't think this is useful (the least classes we have the better). The issue you showed is a theme issue. There is a rule which says:
```less
#wrapwrap .s_four_columns_fw div {
    height: 100%;
}
```
which is wrong.
It should be more precise so that the 100% rule applies only on the own snippet div elements (`> .container-fluid, .container-fluid > .row, .container-fluid > .row > div`).